drivers/bus/pci/pci.h

```diff
@@ -105,12 +108,31 @@ typedef struct _PCI_DRIVER_EXTENSION
     KSPIN_LOCK BusListLock;
 } PCI_DRIVER_EXTENSION, *PPCI_DRIVER_EXTENSION;
 
+typedef struct _PCI_TYPE1_CFG_CYCLE_BITS
+{
+    union
+    {
+        struct
+        {
+            ULONG InUse:2;
+            ULONG RegisterNumber:6;
+            ULONG FunctionNumber:3;
+            ULONG DeviceNumber:5;
+            ULONG BusNumber:8;
+            ULONG Reserved2:8;
+        } bits;
+        ULONG AsULONG;
+    } u;
+} PCI_TYPE1_CFG_CYCLE_BITS, *PPCI_TYPE1_CFG_CYCLE_BITS;
```

Comment:
This is a private structure between HAL and PCI driver, right? Let's just simplify it:
```suggestion
typedef union _PCI_TYPE1_CFG_CYCLE_BITS
{
    struct
    {
        UINT32 InUse:2;
        UINT32 RegisterNumber:6;
        UINT32 FunctionNumber:3;
        UINT32 DeviceNumber:5;
        UINT32 BusNumber:8;
        UINT32 Reserved2:8;
    };
    UINT32 AsULONG;
} PCI_TYPE1_CFG_CYCLE_BITS, *PPCI_TYPE1_CFG_CYCLE_BITS;
```

Reply:
I suppose their definition come literally from a PCI standard reference manual or sthg. Also, why doing the simplification just for that one, and not for the other `PCI_***_BITS` structures as well for consistency?